presto-docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -0,0 +1,25 @@
+====================
+Black Hole Connector
+====================
+
+The Black Hole connector works in similar way as /dev/null device file in Linux operating system. 
```

Comment:
Sorry for being vague. This was a note for us to fix it up. We do this for all documentation changes (even if written by a native speaker).